src/utilities.cc

```diff
@@ -304,8 +305,10 @@ void reportEndpoint(uv_handle_t* h, struct sockaddr* addr, const char* prefix,
   uv_getnameinfo_t endpoint;
   if (uv_getnameinfo(h->loop, &endpoint, nullptr, addr, NI_NUMERICSERV) == 0) {
 #ifdef __MVS__
-    __e2a_s(endpoint.host);
-    __e2a_s(endpoint.service);
+    if (__isASCII() == 0) {
```

Comment:
https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.3.0/com.ibm.zos.v2r3.ceev100/isascii.htm suggests we should `#include <ctype.h>` somewhere?

Reply:
That doc applied to z/OS v2.1; on v2.2, `#include <ctype.h>` doesn't find __isASCII(); I `#include <_Nascii.h>` directly, where __isASCII() is defined.